components/scream/src/share/scream_pack.hpp

```diff
@@ -367,6 +375,24 @@ OnlyPack<Pack> range (const typename Pack::scalar& start) {
   return p;
 }
 
+#undef scream_pack_gen_assign_op_p
```

Comment:
I undef'ed these header impl macros to avoid (largely theoretical since the macro names are long and specific) pollution of namespace.